_posts/0000-01-02-Sophiehwl2177.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
“The best thing about the future is that it comes one day at a time.”
– Abraham Lincoln